src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExperimentalOptions.cs

```diff
@@ -27,10 +29,20 @@ public ExperimentalOptions(IConfiguration configuration)
         {
             this.EmitLogEventAttributes = emitLogEventAttributes;
         }
+
+        if (configuration.TryGetBoolValue(EnableInMemoryRetryEnvVar, out var enableInMemoryRetry))
+        {
+            this.EnableInMemoryRetry = enableInMemoryRetry;
+        }
     }
 
     /// <summary>
-    /// Gets or sets a value indicating whether log event attributes should be exported.
+    /// Gets a value indicating whether log event attributes should be exported.
+    /// </summary>
+    public bool EmitLogEventAttributes { get; private set; } = false;
+
+    /// <summary>
+    /// Gets a value indicating whether retries should be enabled in case of transient errors.
     /// </summary>
-    public bool EmitLogEventAttributes { get; set; } = false;
+    public bool EnableInMemoryRetry { get; private set; } = false;
```

Comment:
nit: You probably don't need `private set` or `= false`.

```csharp
public bool EmitLogEventAttributes { get; }
public bool EnableInMemoryRetry { get; }
```

Because...

1) Only seems to be set in the ctor.
2) `false` is the default value for `bool`.